src/Catalog/Helpers/Retry.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+
+namespace NuGet.Services.Metadata.Catalog.Helpers
+{
+    public class Retry
```

Comment:
Can Polly be used? If not maybe add comments why not.

Reply:
This follows the [pattern](https://github.com/NuGet/NuGet.Services.Metadata/blob/master/src/NuGet.Indexing/Retry.cs) we already have in NuGet.Services.Metadata. Pulling an external library for a tool that we'll run for a little while and discard seems a bit excessive.

Reply:
I tried pulling in Polly, but unfortunately it depends on netstandard pulling great deal of System.* libraries into the output directory and causes a lot of new warnings.